extension/options.html

```diff
@@ -29,6 +29,12 @@
 		</div>
 	</div>
 	
+	<h2>Default Format</h2>
+	<p>Select the format from the table below that should be used when the keyboard shortcut is activated.</p>
+	<!-- when table row is selected, show selection below  -->
+	<p>Default is currently <em id="current-default-format">{{defaultFormatName}}</em></p>
+	<button id="set-default-format"></button>
+
```

Comment:
Any better verbiage for explaining this to users?
![image](https://cloud.githubusercontent.com/assets/6391742/16213779/446cb55c-370f-11e6-9380-52bfd280ead4.png)